src/python/pants/backend/python/tasks2/python_binary_create.py

```diff
@@ -103,6 +103,13 @@ def _create_binary(self, binary_tgt, results_dir):
 
       builder = PEXBuilder(path=tmpdir, interpreter=interpreter, pex_info=pex_info, copy=True)
 
+      if binary_tgt.shebang:
+        self.context.log.info("Found Python binary target {} with customized shebang, using it: {}"
```

Comment:
Use `"` or `'` consistently when you can, we generally default to `'` and it looks like that works here and is what you use in the debug logger below.

Reply:
Valid point, done.